isb/Dockerfile

```diff
@@ -1,5 +1,22 @@
 # syntax=docker/dockerfile:1
-FROM python:latest
+
```

Comment:
This takes advantage of Docker's multi-stage builds feature. https://docs.docker.com/develop/develop-images/multistage-build/